### DISCUSSION TITLE
昆蟲影響大氣電荷

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#858] [General]
https://udn.com/news/story/6897/6718140

很多昆蟲能夠藉由感應電場變化遷徙覓食等等，藉由揮動翅膀與空氣摩擦產生電荷並形成電場，而研究發現成群結隊的蜜蜂竟能提高大氣電流100~1000伏特，如果之後發現昆蟲真的能夠影響天氣，那或許是不是能夠利用昆蟲產生風暴雲，解決各地缺水的問題?

**Comment by moonnight3503** [comment]
太酷了!昆蟲竟然能夠在飛行時透過翅膀和空氣摩擦攜帶電荷，還能利用電廠進行交流，我以為蜜蜂交流只會用8字舞的方式，哈哈!

**Comment by Kotori0501** [comment]
如果昆蟲能夠影響天氣
對研究天氣的人來說，又多了一個領域可以研究
感覺挺酷的

**Comment by Chiway1229** [comment]
原來這個事情真的證實了蝴蝶效應
昆蟲微小的翅膀振動能導致龐大的事件連鎖發生
說不定我現在的一舉一動都能某種程度的影響未來也說不定

**Comment by peggy0329** [comment]
這篇文章好有趣，蜜蜂居然可以透過飛行時翅膀和空氣摩擦攜帶電荷
可以得知其實萬物都會一點一點的影響世界
這個研究涉及生物、物理及地科
希望結合更領域的專家進行研究，或許這個發現可以為社會帶來幫助

**Comment by Zoe06060606** [comment]
我想蝴蝶效應就是這樣產生的吧 這個新領域在未來應該也會是相當前衛的主題吧

**Comment by williamyen0623** [comment]
大自然的力量遠超乎我們的認知，尚有很多我們未發先的奧秘就藏在我們常看見的自然事物中

**Comment by Evan0979** [comment]
這就是所謂的蝴蝶效應嗎,大自然真的好神奇!!

**Comment by StevenWanglolz** [comment]
這研究也太有趣了吧!好期待未來在現實生活中的應用!看到其他人都留言蝴蝶效應我才想到，這些小昆蟲動翅膀真的可以有很大的影響!

**Comment by CPCpyt** [comment]
這個猜想非常酷！這就是蝴蝶效應的加強版嗎！
不過這要實際實驗和運用我想需要很多的想像力以及...經費！
